### PR TITLE
Respect user wish to leave existing VM intact

### PR DESCRIPTION
**Pull request by jhutar**

### CODE REVIEW

**Comment by jhutar** [comment]
Tested with running `snap-guest -b rhel76-base -t test-vm -m 2048 -c 2` few times and responding:
 * "" (just pressed Enter) - VM not deleted
 * "n" - VM not deleted
 * "Y" - VM not deleted
 * "y" - VM deleted

File: snap-guest [hunk 1]
```diff
@@ -219,13 +219,18 @@ if [ -f "$TARGET_IMG" ]; then
   if [ $FORCE -eq 1 ]; then
     REPLY="y"
   else
-    read -p "Shutdown and remove host with this name first (y/[n])? "
+    read -p "Shutdown and remove host with this name first (y/[n])? " REPLY
```
Comment (lpramuk) [inline snap-guest:222]: ```If  no  names  are supplied, the line read is assigned to the variable REPLY.```

REPLY is not needed

**Review by lpramuk — APPROVED** [review]
ACK pending nitpicks :-)

File: snap-guest (hunk 1 above)
Comment (jhutar) [reply snap-guest:222]: Hmm, did not knew that, thank you @lpramuk! Anyway, "Explicit is better than implicit" ;-)

**Pull request merged**